src/lib/NonlinearDutchDecayLib.sol

```diff
@@ -39,7 +40,7 @@ library NonlinearDutchDecayLib {
             return startAmount.bound(minAmount, maxAmount);
         }
 
-        uint16 blockDelta = uint16(block.number - decayStartBlock);
```

Comment:
add comment here about rationale for the change, and the expected behavior in case of overflow